tests/trinity/core/integration_test_helpers.py

```diff
@@ -96,3 +109,57 @@ class FakeAsyncChain(MiningChain):
     coro_import_block = coro_import_block
     coro_validate_chain = async_passthrough('validate_chain')
     coro_validate_receipt = async_passthrough('validate_receipt')
+    chaindb_class = FakeAsyncChainDB
+
+
+class ByzantiumTestChain(FakeAsyncChain):
+    vm_configuration = ((0, ByzantiumVM),)
+    network_id = 999
+
+
+FUNDED_ACCT = keys.PrivateKey(
+    decode_hex("49a7b37aa6f6645917e7b807e9d1c00d4fa71f18343b0d4122a4d2df64dd6fee"))
+
+
+def load_mining_chain(db):
+    class POWByzantiumVM(POWMiningMixin, ByzantiumVM):
+        pass
+
+    GENESIS_PARAMS = {
+        'coinbase': constants.ZERO_ADDRESS,
+        'difficulty': 5,
+        'gas_limit': 3141592,
+        'timestamp': 1514764800,
+    }
+
+    GENESIS_STATE = {
+        FUNDED_ACCT.public_key.to_canonical_address(): {
+            "balance": 100000000000000000,
+        }
+    }
+
+    return build(
+        FakeAsyncChain,
+        fork_at(POWByzantiumVM, 0),
+        genesis(db=db, params=GENESIS_PARAMS, state=GENESIS_STATE),
+    )
+
+
+class DBFixture(Enum):
+    twenty_pow_headers = '20pow_headers.ldb'
+
+
+def load_fixture_db(db_fixture, db_class=LevelDB):
+    """
+    Extract the database from the zip file to a temp directory, which has two benefits and one cost:
+    - B1. works with xdist, multiple access to the ldb files at the same time
+    - B2. prevents dirty-ing the git index
+    - C1. slows down test, because of time to extract files
+    """
+    assert isinstance(db_fixture, DBFixture)
+    zip_filename = (db_fixture.value + '.zip')
+    zipped_path = Path(__file__).parent.parent / 'integration' / 'fixtures' / zip_filename
```

Comment:
I'd recommend we place the first part of this as a top-level constant in this module.  With it embedded in the function it seems easier to forget to update it if this were to move somewhere else.